fe/fe-core/src/main/java/com/starrocks/lake/StarMgrMetaSyncer.java

```diff
@@ -217,6 +205,58 @@ private void deleteUnusedShardAndShardGroup() {
         }
     }
 
+    private void cleanOneGroup(long groupId, StarOSAgent starOSAgent, List<Long> emptyShardGroup) {
+        try {
+            List<Long> shardIds = starOSAgent.listShard(groupId);
+            if (shardIds.isEmpty()) {
+                emptyShardGroup.add(groupId);
+                return;
+            }
+            // delete shard from star manager only, not considering tablet data on be/cn
+            if (Config.meta_sync_force_delete_shard_meta) {
+                forceDeleteShards(groupId, starOSAgent, shardIds);
+            } else {
+                // drop meta and data
+                long start = System.currentTimeMillis();
+                dropTabletAndDeleteShard(shardIds, starOSAgent);
+                LOG.debug("delete shards from starMgr and FE, shard group: {}, cost: {} ms",
+                        groupId, (System.currentTimeMillis() - start));
+            }
+        } catch (Exception e) {
+            LOG.warn("delete shards from starMgr and FE failed, shard group: {}, {}", groupId, e.getMessage());
+        }
+    }
+
+    private static void forceDeleteShards(long groupId, StarOSAgent starOSAgent, List<Long> shardIds) {
+        LOG.debug("delete shards from starMgr only, shard group: {}", groupId);
+        // before deleting shardIds, let's record the root directory of this shard group first
+        // root directory has the format like `s3://bucket/xx/db15570/15648/15944`
+        String rootDirectory = null;
+        long shardId = shardIds.get(0);
+        try {
+            // all shards have the same root directory
+            ShardInfo shardInfo = starOSAgent.getShardInfo(shardId, StarOSAgent.DEFAULT_WORKER_GROUP_ID);
+            if (shardInfo != null) {
+                rootDirectory = shardInfo.getFilePath().getFullPath();
+            }
+        } catch (Exception e) {
+            LOG.warn("failed to get shard root directory from starMgr, shard id {}, {}", shardId,
+                    e.getMessage());
+        }
+        try {
+            starOSAgent.deleteShards(new HashSet<>(shardIds));
+            if (StringUtils.isNotEmpty(rootDirectory)) {
+                LOG.info(
+                        "shard group {} deleted from starMgr only, you may need to delete remote file path manually," +
+                                " file path is: {}", groupId, rootDirectory);
+            } else {
```

Comment:
remove this else, log is already printed in line 243

Reply:
done